js_modules/dagit/packages/core/src/asset-graph/useAssetGraphData.tsx

```diff
@@ -38,6 +38,7 @@ export type AssetGraphQueryItem = GraphQueryItem & {
 export function useAssetGraphData(opsQuery: string, options: AssetGraphFetchScope) {
   const fetchResult = useQuery(ASSET_GRAPH_QUERY, {
     notifyOnNetworkStatusChange: true,
+    fetchPolicy: 'cache-first', // todo bengotow: Is this correct?
```

Comment:
I think this can change to cache-and-network, I don't think the choice was explicit and it'd be nice for it to refresh as necessary based on the latest data